source/main/index.js

```diff
@@ -128,8 +129,8 @@ const onAppReady = async () => {
 
   logger.info('Updating System-info.json file', { ...systemInfo.data });
 
-  // We need DAEDALUS_INSTALL_DIRECTORY in PATH
-  // in order for the cardano-launcher to find wallet and node bins
+  // We need DAEDALUS_INSTALL_DIRECTORY in PATH in order for the
+  // cardano-launcher to find cardano-wallet and cardano-node executables
   process.env.PATH = [
     process.env.PATH,
     process.env.DAEDALUS_INSTALL_DIRECTORY,
```

Comment:
@rvl I would appreciate your help here... It seems the installers are not setting this and this is why we do it in Daedalus.
It is awkward and I don't like it but that is the only way it works at the moment. Feel free to help us out and update this part of the code (or the installers) if you have time 🙏

Reply:
If there are still problems starting `cardano-node.exe` on windows (because we actually need `cardano-node.exe` to be spawned with the state directory as its cwd), then check what the working directory is for the Daedalus process.

Reply:
It fails to start on OSX too.

Reply:
I pushed a change. Would it be possible to get some help testing to see how much still works (especially on windows)?

Reply:
Sure. @dmitrii-gaico help us out here 🙏

Reply:
Our OSX setup is more interesting than I expected.
Anyway, I pushed a fix for that.
I could only test it in a simulated installation environment, not an actual macOS install.

Reply:
@dmitrii-gaico please help us with testing today / tomorrow 🙏